server/app/routes/users/index.js

```diff
@@ -25,6 +27,24 @@ router.get('/', function(req, res, next) {
     .catch(next);
 })
 
+router.post('/', function(req, res, next){
+    Users.create(req.body)
```

Comment:
I guess there is not verification happening - no check for admin privileges - just wanted to point it out , no need to put that in now.